lib/getMermaidDiagram.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Generates mermaid class diagram from default output syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default output syntax
+ * @returns {String} class diagram following mermaid syntax
+ */
+function getMermaidDiagram(metrics) {
+  const visitedChannels = [];
+  const visitedServices = [];
+  let i=1;
+  let mermaidSyntax = '';
+  metrics.forEach((relations,server) => {
+    const [url, protocol] = server.split(',');
+    mermaidSyntax+= `\nclass server${i} { \n url: ${url} \n protocol: ${protocol}\n}`;
+
+    relations.forEach((operations, channel) => {
+      const channelName = channel.split(/[/{}]/).join('');
+      
+      if (!visitedChannels.includes(channelName)) {
+        visitedChannels.push(channelName);
+        mermaidSyntax+= `\nclass ${channelName} { \n // TODO: metadata to be added \n}\n`;
```

Comment:
We can change it to:

```suggestion
        mermaidSyntax+= `\nclass ${channelName} { \n metadata \n}\n`; // TODO: add metadata
```